lib/IDE/CodeCompletionResultBuilder.h

```diff
@@ -15,30 +15,23 @@
 
 #include "swift/AST/Types.h"
 #include "swift/Basic/LLVM.h"
-#include "swift/Basic/StringExtras.h"
 #include "swift/IDE/CodeCompletionResult.h"
 #include "swift/IDE/CodeCompletionResultSink.h"
-#include "swift/Parse/Lexer.h"
-#include "llvm/ADT/SmallVector.h"
-#include "llvm/ADT/StringRef.h"
-#include "llvm/ADT/StringSwitch.h"
+#include "swift/IDE/CodeCompletionStringBuilder.h"
 
 namespace clang {
 class Module;
-}
+} // namespace clang
 
 namespace swift {
 class Decl;
 class ModuleDecl;
 
 namespace ide {
 
-class CodeCompletionStringPrinter;
-
 class CodeCompletionResultBuilder {
-  friend CodeCompletionStringPrinter;
-  
   CodeCompletionResultSink &Sink;
+  CodeCompletionStringBuilder StringBuilder;
```

Comment:
Sorry for not suggesting this sooner, but could we have `CodeCompletionResultBuilder` inherit from `CodeCompletionStringBuilder`? That way we don't need to deal with the awkwardness of sometimes having to call a method on the string builder and sometimes not

Reply:
Done. Please recheck 🙏🏼